docs/UserGuide.md

```diff
@@ -190,3 +190,66 @@ Action | Format, Examples
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
 **Help** | `help`
+
+### Editing an event : `eedit`
+
+Edits an existing event in the event list.
+
+Format: `eedit INDEX [n/NAME] [at/START_TIME] [end/END_TIME] [d/DESCRIPTION] [a/ADDRESS] [z/ZOOM] [dt/TAG_DELETED]…​ [t/TAG_ADDED]…​`
+
+Take note of the following time format:
+Start time should be of format “dd-MM-yyyy HH:mm” (date-month-year Hour:minutes in 24 hr format).
+
+* `elist` followed by `eedit` 2 edits the 2nd event from the event list.
+* Edits the event at the specified `INDEX`. `INDEX` refers to the index number shown in the displayed event list. `INDEX` **must be a positive integer** eg 1, 2, 3, …​
+* **At least one** of the optional fields must be provided.
+* Existing values will be updated to the input values.
+* use `t/` to add a tag.
+* use `dt/` to remove a tag.
+* use `dt/*` to remove **all** tags
+* When editing tags, the tags to be deleted will be removed first, before new tags are added
+
+Examples:
+* `elist` followed by `eedit 2 n/CS2103T Exam dt/Easy_exams t/Hard_exams` changes the name and tag of the 2nd event on the event list to `CS2103T Exam` and `Hard_exams` respectively.
+* `efind Betsy` followed by `eedit 2 n/Betsy’s Wedding dt/*` edits the name of the 2nd event from the results of the `efind` command to be `Betsy’s Wedding` and clears all existing tags.
+* `eedit 3 dt/TA`  deletes the `TA` tag from the 3rd event.
+* `eedit 4 dt/*` deletes all tags from the 4th event.
+* `eedit 5 dt/Easy_exams t/Hard_exams` first deletes the `Easy_exams` tag, then adds `Hard_exams` tag to the 5th event.
+
+### Locating persons by name : `efind`
+
+Finds events which contain any of the given keywords.
+
+Format: `efind KEYWORD [MORE_KEYWORDS]`
+
+* **case-insensitive**. e.g `exams` will match `Exams`
+* order of the keywords does not matter. e.g. `Exam Hard` will match `Hard Exam`
```

Comment:
Start with capital letters. For line 255, consider writing "Keyword is case-insensitive".